src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -608,10 +613,21 @@ public Task<DocumentDatabase> TryGetOrCreateResourceStore(StringSegment database
                         if (database.IsCompletedSuccessfully)
                             database.Result.LastAccessTime = database.Result.Time.GetUtcNow();
 
+                        if (_logger.IsOperationsEnabled)
+                            _logger.Operations(database.IsCompletedSuccessfully
+                                ? $"Successfully loaded database '{databaseName.Value}' from cache. Action taken by '{caller}'."
+                                : $"Attempted to load database '{databaseName.Value}' from cache. Action taken by '{caller}'.");
+
                         return database;
                     }
                 }
-                return CreateDatabase(databaseName, wakeup, ignoreDisabledDatabase, ignoreBeenDeleted, ignoreNotRelevant, caller, addToInitLog);
+
+                var createdDatabase = CreateDatabase(databaseName, wakeup, ignoreDisabledDatabase, ignoreBeenDeleted, ignoreNotRelevant, caller, addToInitLog);
+
+                if (_logger.IsOperationsEnabled)
```

Comment:
aren't you introducing here a spam of logs per request? I think that we just want to know when database is actually being created, so when we are creating the task which is being added to the concurrent dict

Reply:
Oh, you're right. Now I have applied a different approach that elevates part of the existing initializing logs to the Operations level, now displaying the start and completion of database loading at the Operations level.